recipes/miniFASTA/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "miniFasta" %}
+{% set version = "2.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/not-a-feature/miniFASTA/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fe83656e252306ec9683efc90ece2f20fcf78806d285b9056a02096ecb120e12
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - miniFasta
+
+about:
+  home: https://github.com/not-a-feature/miniFASTA
+  license: GPL-3.0
```

Comment:
As the linter said, this license is not acceptable. Can you please clarify whether you are using `GPL-3.0-only` or `GPL-3.0-or-later`?

Ref: https://spdx.org/licenses/

Reply:
Good point. I changed that.